plugins/inputs/statsd/statsd.go

```diff
@@ -283,9 +285,11 @@ func (s *Statsd) udpListen() error {
 				log.Printf("ERROR READ: %s\n", err.Error())
 				continue
 			}
+			bufCopy := make([]byte, n)
```

Comment:
If people are hitting this listener at a high enough rate to expose the previous race condition, you might want to prefer a `sync.Pool` of byte slices to reduce allocations and garbage. But, I'm just speculating, and writing a benchmark will be more informative.